tests/Rules.Framework.Tests/Evaluation/MultiplicityEvaluatorTests.cs

```diff
@@ -0,0 +1,56 @@
+namespace Rules.Framework.Tests.Evaluation
+{
+    using System;
+    using System.Collections.Generic;
+    using FluentAssertions;
+    using Rules.Framework.Core;
+    using Rules.Framework.Evaluation;
+    using Xunit;
+
+    public class MultiplicityEvaluatorTests
+    {
+        public static IEnumerable<object[]> SuccessCombinations => new[]
```

Comment:
you may add 'readonly' for codacy

Reply:
I did not find this one on Codacy, did you mean to point out another file?

Reply:
I had this problem with my pull request before
![image](https://user-images.githubusercontent.com/1099060/205311676-f5641170-53e9-4603-a05e-fd5716194d0e.png)

Reply:
Oh, understood your point, but in this case, it is a property that only exposes a getter (I know that the difference is subtle and it doesn't go easy with our eyes 😄).

![image](https://user-images.githubusercontent.com/20577216/205321021-97dbaeba-f06d-4eea-841d-4493106bf1b6.png)